website/react-magma-docs/src/pages/api/pagination.mdx

```diff
@@ -71,10 +71,10 @@ export function Example() {
 ```tsx
 import React from 'react';
 
-import { Pagination } from 'react-magma-dom';
+import { Pagination, PageButtonSize } from 'react-magma-dom';
 
 export function Example() {
-  return <Pagination size="large" count={6} page={2} />;
+  return <Pagination size={PageButtonSize.large} count={6} defaultPage={2} />;
```

Comment:
I think we need to update these examples ensuring that:
- Controlled => no defaultPage, page is optional
- Uncontrolled => defaultPage is optional, page is required

This needs to work properly for both Simple and Classic pagination